### ISSUE TITLE
Unused XZ decompression methods in Utils?

### ISSUE THREAD

**Issue by dgelessus** [ATLauncher/ATLauncher#299]
IntelliJ says that `Utils.unXZPackFile` is not used anywhere, and that the `unXZFile`, `unpackFile` and `readFile` are only used (directly or indirectly) from `unXZPackFile`. Would these be safe to remove, or is XZ decompression necessary for anything in particular? The launcher compiles and runs fine without these methods, as far as I can tell.

This is also the only place where the XZ library (`org.tukaani:xz:1.5`) is used in the launcher code, so that could be removed together with these methods. (Not that the 100 KB jar file would make any noticeable difference. :P)

**Comment by RyanTheAllmighty** [comment]
From what I can remember it was added in because Forge dependencies used it.

I mean it looks unused, so I'd say take it out and should be fine

**Comment by dgelessus** [comment]
Hm, I noticed there was a comment above one of the methods pointing to http://atl.pw/1 (nice URL, lol), which redirects to a broken link (the file is [here](https://github.com/MinecraftForge/Installer/blob/master/src/main/java/net/minecraftforge/installer/DownloadUtils.java) currently) in the Forge installer repo. So I suppose it had something to do with Forge at some point. Perhaps some of Forge's libraries were XZ-packed before?

I don't know the launcher code well enough to know where it could have been used. I checked the Git log near the commit that added the XZ methods, but didn't see anything obvious related to Forge that might have used them.
